doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2879,7 +2936,7 @@ NCSDK-18746: The :ref:`liblwm2m_carrier_readme` library fails to complete non-se
 .. rst-class:: v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-3-1 v1-2-1 v1-2-0
 
 NCSDK-12912: The :ref:`liblwm2m_carrier_readme` library does not recover if initial network connection fails
-  When the device is switched on, if :cpp:func:`lte_lc_connect()` returns an error at timeout, it will cause :cpp:func:`lwm2m_carrier_init()` to fail.
+  When the device is switched on, if :c:func:`lte_lc_connect` returns an error at timeout, it will cause ``lwm2m_carrier_init()`` to fail.
```

Comment:
```suggestion
  When the device is switched on, if the :c:func:`lte_lc_connect` function returns an error at timeout, it will cause ``lwm2m_carrier_init()`` to fail.
```